concept/ConceptManager.java

```diff
@@ -73,12 +74,10 @@ public AttributeType getRootAttributeType() {
     }
```

Comment:
The `concept` changes are pretty much entirely just renaming due to updated `protocol`.